Source/WebKit/Platform/spi/Cocoa/AuthenticationServicesCoreSPI.h

```diff
@@ -32,6 +32,8 @@
 #else
 @interface ASCWebKitSPISupport : NSObject
 @property (class, nonatomic) BOOL shouldUseAlternateCredentialStore;
++ (BOOL)arePasskeysDisallowedForRelyingParty:(nonnull NSString *)relyingParty;
++ (void)getArePasskeysDisallowedForRelyingParty:(nonnull NSString *)relyingParty withCompletionHandler:(nonnull void (^)(BOOL))completionHandler;
```

Comment:
You don't seem to use the `getArePasskeysDisallowedForRelyingParty` method, so may not be necessary to forward-declare it.